zerver/lib/emoji_conversions.py

```diff
@@ -0,0 +1,13 @@
+emoji_conversions = {
+    ':)': ':simple_smile:',
+    ':-)': ':simple_smile:',
+    ';)': ':wink:',
+    ';-)': ':wink:',
+    ':(': ':disappointed:',
+    ':-(': ':disappointed:',
+    ';(': ':cry:',
+    ';-(': ':cry:',
```

Comment:
I guess `;(` or `;-(` are not the right `:cry:` emoticons. Here at [Wikipedia](https://en.wikipedia.org/wiki/List_of_emoticons), `:'(` and `:'-(` are considered as `:cry:` emoticons.